ignite/pkg/cosmosutil/genesis/genesis.go

```diff
@@ -0,0 +1,94 @@
+package genesis
```

Comment:
Should we move the contents of [this](https://github.com/ignite-hq/cli/blob/a84f7b8/ignite/pkg/cosmosutil/genesis.go) to here?

Reply:
this file was deleted in this PR

Reply:
got it